tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -65,16 +65,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Add Renewable Formulation"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Load Factors - Describing the Electricity Sector\n",
-    "#### Peak load \n",
-    "If we further specify our demand, we will find that the demand for electricity (!not the demand for useful light but for electricity!) has a peak load that is different from the annual average load; e.g: if annual average load is 5 GWh peak load during midday might be 10GW --> the peak load factor is equal to 2."
+    "## Improving the Representation of Electricity Sector\n",
+    "### Peak load factor\n",
+    "The input demand for electricity (notice: not the demand for useful light but for electricity) in the model shows the average electricity demand in the given time (which in our example is a year). However, power systems need enough installed capacity not only to cover the average electricity demand but also to meet the peak demand, i.e., maximum load throughout that time. This feature can be specified in the model using parameter `peak_load_factor`. For example, if annual average load is 5 GWa peak load amy be 10GW, hence, the peak load factor is equal to 2."
```

Comment:
“For example, if annual average load is 5 GW, the peak load may be twice as high, at 10 GW; hence, the peak load factor is equal to 2.”

Reply:
Thanks, reads better now.